### PR TITLE
updated the react folder_structure

### PR DESCRIPTION
**Pull request by rakshith6404**
I have created an issue on the Latest folder_structure regarding the issue https://github.com/plone/training/issues/727 and also have added using the tree command.

I created a new branch as i was getting conflict errors , Looking forward for a review for the same.

@stevepiercy , what if we could add a note saying and telling about them about there are many files and folders inside the node-modules folder?  

Screenshot from 2023-03-07 19-51-55

### CODE REVIEW

File: docs/react/bootstrap.md [hunk 1]
```diff
@@ -43,22 +43,26 @@ It will create a folder called `my-app` inside the current folder with the follo
 
 ```console
 my-app
-├── README.md
 ├── node_modules
 ├── package.json
-├── .gitignore
```
Comment (stevepiercy) [inline docs/react/bootstrap.md:49]: Please restore `.gitignore`. It might be invisible, but it definitely exists.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by stevepiercy** [comment]
> @stevepiercy , what if we could add a note saying and telling about them about there are many files and folders inside the node-modules folder?

Good idea. Suggest:

```
It will create a folder called `my-app` inside the current folder with the following structure.
We omit from the structure the files and directories inside the `node_modules` directory because there are too many to list.
```

**Comment by rakshith6404** [comment]
I have updated the folder structure and also have added the note required which @stevepiercy have suggested.

File: docs/react/bootstrap.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ It will create a folder called `my-app` inside the current folder with the follo
 ```console
 my-app
 ├── node_modules
+├── .gitignore
```
Comment (stevepiercy) [inline docs/react/bootstrap.md:47]: Fix sort order.

```suggestion
├── .gitignore
├── node_modules
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: docs/react/bootstrap.md (hunk 2 above)
Comment (rakshith6404) [reply docs/react/bootstrap.md:47]: This was the order i got from when using ```tree``` command

File: docs/react/bootstrap.md [hunk 3]
```diff
@@ -43,22 +43,31 @@ It will create a folder called `my-app` inside the current folder with the follo
 
 ```console
 my-app
-├── README.md
+├── .gitignore
 ├── node_modules
```
Comment (stevepiercy) [inline docs/react/bootstrap.md:47]: Let's delete this, so that it aligns with the note.

```suggestion
```

File: docs/react/bootstrap.md (hunk 2 above)
Comment (stevepiercy) [reply docs/react/bootstrap.md:46]: That is... strange. What command do you use?

Try the following to go 2 levels deep, show all files including invisibles, and ignore the `node_modules` directory.

```
tree -L 2 -a -I node_modules
```

Comment (stevepiercy) [reply docs/react/bootstrap.md:46]: In my output, `.` files are first, then files and directories that start with an uppercase letter, then files and directories that start with a lowercase letter.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: docs/react/bootstrap.md (hunk 2 above)
Comment (rakshith6404) [reply docs/react/bootstrap.md:46]: It was mistake that .gitignore was before the node_modules....i have updates the mistake...pls review it

**Comment by rakshith6404** [comment]
This is what i get when i exclude the node_module folder using ```tree -I "node_modules\```": 
Screenshot from 2023-03-08 16-24-27

File: docs/react/bootstrap.md (hunk 3 above)
Comment (rakshith6404) [reply docs/react/bootstrap.md:47]: > Let's delete this, so that it aligns with the note.

Could you tell me where should the alignment be with the note?

**Review by stevepiercy — APPROVED** [review]

File: docs/react/bootstrap.md (hunk 3 above)
Comment (stevepiercy) [reply docs/react/bootstrap.md]: The note says we omit it. I committed the suggestion. This is good to merge. Thank you!

**Pull request merged**